src/field_32bit.rs

```diff
@@ -30,14 +30,11 @@ use core::ops::Neg;
 
 use subtle::ConditionallyAssignable;
 
-use utils::{load3, load4};
-
 /// A `FieldElement32` represents an element of the field GF(2^255 - 19).
 ///
 /// In the 32-bit implementation, a `FieldElement32` is represented in
-/// radix 2^25.5 as ten `i32`s, so that an element t, entries
-/// t[0],...,t[9], represents the integer t[0]+2^26 t[1]+2^51
-/// t[2]+2^77 t[3]+2^102 t[4]+...+2^230 t[9].
+/// radix 2^25.5 as ten `u32`s, so that an element t, entries
+/// t[0],...,t[9], represents `sum(t[i]*2^ceil(i*51/2))`.
```

Comment:
This comment doesn't actually get clearer (IMHO) unless you already understand that the radix is alternating between 25 and 26 bits.